### ISSUE TITLE
The RFI prior's default width is a maximum where the key says typical

### ISSUE THREAD

**Issue by chrisfinlay** [epfl-radio-astro/tabascal#227]
#221 renamed `rfi.var` to `rfi.std`, meaning the RFI's typical `rms|V|` in Jy. The default that fills it when the key is null was preserved exactly, and it does not mean that.

## What it is

```python
rfi_config["std"] = _LATENT_POWER * float(jnp.max(jnp.abs(vis_obs)))
```

Twice the **largest** visibility in the observation. The `_LATENT_POWER` is there only to reproduce the prior the old key set — `rfi.var` defaulted to `max|vis_obs|` and meant half what `rfi.std` means, so the doubling keeps every existing run identical. That part is correct and deliberate.

What is left over is the statistic. The key now says *typical* and the default supplies a *maximum*, doubled. On an observation with a bright pass, the maximum is far above the typical amplitude, so the default prior is wider than the RFI it is a prior on — and a prior wider than its signal is what #218 measured the cost of on the astronomical side, where matching the width to the sky was worth a third of the error.

## Why it was not changed under the rename

#221 was scoped as rename and translation, and every shipped configuration takes this default, so changing the statistic would have changed the prior of every pipeline reference under a change whose whole claim was that nothing moves.

## What to work out

- What the right statistic is. `rms|V|` over the whole observation includes the RFI-free stretches and would come out well below the RFI's own amplitude; the RFI is a transient, so its typical amplitude *while present* is the quantity the key names.
- Whether the elevation mask helps: the model already knows which times each satellite is above the horizon, so an rms over the in-view samples is available at setup for the same cost.
- Whether it should reuse `ast.pow_spec.std: data`'s machinery, which measures `rms|V|` per baseline over unflagged samples (#220 tracks making that RFI-aware from the other direction — the two may want the same estimator).
- Whether the default should be per baseline at all, as the astronomical one is, or stay a scalar.

## Acceptance

- [ ] The default is a statistic the key's name describes
- [ ] Measured against the shipped 8A simulation, whose true RFI `rms|V|` is known (10.3-11.6 Jy per baseline)
- [ ] Pipeline references re-recorded, since this does change the prior

## Related

- #221 — the rename that made the mismatch visible
- #220 — the astronomical estimator's own contamination problem, from the other side
- #218 — what a mismatched prior width costs

**Comment by chrisfinlay** [comment]
Measured on the shipped 8A simulation, which is the acceptance criterion above.

| | per baseline |
|---|---|
| **default `rfi.std`** = `2 * max\|vis_obs\|` | **46.45 Jy** |
| true RFI `rms\|V\|` | 10.32 – 11.57 (median **11.01**) |
| true sky `rms\|V\|` | 0.27 – 2.90 (median 1.53) |
| observed `rms\|V\|` | 9.50 – 12.50 (median 11.20) |

So the default is **4.2x the RFI it is a prior on**.

## The estimator that would fix it is already in the tree

The observed `rms|V|` is **1.02x** the true RFI here — 11.20 against 11.01. Not a coincidence: the RFI dominates the sky by about 7x in this simulation, so `rms|vis_obs|` is very nearly `rms|vis_rfi|`.

That is the same statistic `ast.pow_spec.std: data` computes (`GPVisAst._std_from_data`), which is worth noting because the two estimators want *opposite* things from the same number. The astronomical prior wants the sky and gets the RFI, which is why it warns and why #220 exists. The RFI prior wants the RFI and would get it, on any observation where the RFI is what dominates.

## Where it stops being right

`rms|vis_obs|` is an upper bound on the RFI, since it carries the sky and the noise too. It is nearly exact when the RFI dominates and increasingly wrong as it does not — on an observation with weak RFI it measures the sky and sets the RFI prior far too wide, the same failure `ast.pow_spec.std: data` has in the other direction.

So `rms|vis_obs|` is a much better default than `2 * max|vis_obs|` (1.02x against 4.2x here) without being the right answer everywhere. The RFI-aware version is the same machinery #220 wants: once the model's own view of which samples are contaminated exists, both priors can measure the half of the data they each care about, and neither has to warn.

Measurement: `vis_obs`, `vis_rfi`, `vis_ast` from the 8A zarr, reduced over time and frequency for a per-baseline `rms|V|`, in float64.
